tests/BSCAccountProvider.test.ts

```diff
@@ -0,0 +1,25 @@
+import { makeBSCAccountProvider } from "../src/BSCAccountProvider";
+import { makeCoinGeckoProvider } from "../src/CoinGeckoProvider";
+import { TEST_DATA } from "./const";
+
+const priceProvider = makeCoinGeckoProvider();
+
+describe("BSCAccountProvider", () => {
+  const provider = makeBSCAccountProvider(TEST_DATA.bnbAddress, priceProvider, TEST_DATA.alchemyApiKey);
+
+  test("should return balance as a number", async () => {
+    const balance = await provider.balance;
+    expect(typeof balance).toBe("number");
+  });
+
+  test("should return transfer history as array", async () => {
```

Comment:
please be consistent within each `describe` block, use `it` for all unit-tests, or use `test`, please don't mix `it` and `test`